docs/developer/checkout/problems.mdx

```diff
@@ -4,19 +4,6 @@ title: Problems
 
 ## Introduction
 
-:::info
-
-This feature was introduced in **Saleor 3.15**.
```

Comment:
Is it expected that we will remove the note about the added version?

Reply:
The issue is that only selected features have such labels; if we were to be consistent, then all of the docs would be full of such notes, which is not scalable. 

Perhaps for recent 1-2 version backs, it might be useful, but documenting 5 versions back is quite redundant, IMO. Alternatively, we can add docs release labels for anyone who wants to track back docs.